flint.ui/src/components/gcbm/UploadAndEditFiles.vue

```diff
@@ -174,15 +180,17 @@ export default {
 
     const handleUpload = () => {
       const formData = new FormData()
+      formData.append('title', store.state.gcbm.config.title)
+
       fileList.value.forEach((file) => {
-        formData.append('files[]', file)
+        formData.append(backendFileTypeName, file)
       })
 
       uploadingVars.value.uploading = true
 
-      fetch('https://run.mocky.io/v3/ebca54c1-1021-4356-a5dc-1c5e2330c2c4', {
+      fetch(`${process.env.VUE_APP_REST_API_GCBM}/gcbm/upload/${backendFileTypeName}`, {
```

Comment:
the API endpoint to upload the simulation title is `/upload/title` you don't have to send the file name in the URL. Add the file name to the payload and it should work fine.

Reply:
@SanjaySinghRajpoot Can you please raise a PR for it?

Reply:
ok, I will fix this error.

Reply:
@SanjaySinghRajpoot I've sent you a message on slack, that line is for uploading the files, not the title.
the `upload/title` endpoint is for creating a new simulation correct? That change is in the gcbm.js file: [https://github.com/moja-global/FLINT-UI/pull/347/commits/bb5f5cf0c9486eed8abe3d319afb63692410f64a#diff-7626babda3a6bd8350[…]eb4c647bc9bc2425e79d4R71](https://github.com/moja-global/FLINT-UI/pull/347/commits/bb5f5cf0c9486eed8abe3d319afb63692410f64a#diff-7626babda3a6bd835082ae26ab847abfbd382dc1060eb4c647bc9bc2425e79d4R71)

Reply:
Got it so it will take the dynamic file name from the the `upload/title` and make use of this endpoint. I hope its correct because I don't remember the above endpoint.